docs/docs/cmd/aad/administrativeunit/administrativeunit-member-list.mdx

```diff
@@ -0,0 +1,316 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# aad administrativeunit member list
+
+Retrieves members (users, groups, or devices) of an administrative unit
+
+## Usage
+
+```sh
+m365 aad administrativeunit member list [options]
+```
+
+## Options
+
+```md definition-list
+`-i, --administrativeUnitId [administrativeUnitId]`
+: The id of the administrative unit. Specify either `administrativeUnitId` or `administrativeUnitName`.
+
+`-n, --administrativeUnitName [administrativeUnitName]`
+: The name of the administrative unit. Specify either `administrativeUnitId` or `administrativeUnitName`.
+
+`-m, --memberType [memberType]`
+: When specified, the command returns only objects of specified type. The possible values are `user`, `group` and `device`.
+
+`-p, --properties [properties]`
+: Comma-separated list of properties to retrieve.
+
+`-f, --filter [filter]`
+: OData filter to use to query the list of members with. Applied only when `memberType` is specified.
+```
+
+<Global />
+
+## Remarks
+
+:::info
```

Comment:
Let's not use these elements here. Only use them for remarks that deserve to be highlighted.